basalt/autograd/attributes.mojo

```diff
@@ -83,7 +83,6 @@ struct Attribute(Stringable, CollectionElement):
     var data: Bytes[MAX_DATA_BYTES]
     var data_shape: StaticIntTuple[MAX_RANK]
     var type: AttributeType
-    var size: Int
```

Comment:
This value is necessary, to be able to represent things like statictuple and tensorshape the same way (in the end they are all lists of a value, so they are the same)